trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.slice(0, 10)
```

Comment:
- [ ] Kindly make sure that the output has a maximum of 10 characters. If it's longer it should trim the word.
> As stated in the requirement
 
![image](https://user-images.githubusercontent.com/53402600/227372360-35861fc8-28fe-4f31-b683-35612efbfe71.png)